base/cvd/fastboot/util.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
nit: the original file doesn't have a license, should this file have one?

Reply:
Done